common.vars.pkr.hcl

```diff
@@ -1 +1 @@
-postgres-version = "15.1.1.95"
```

Comment:
On this one, we moved the postgres version to 

https://github.com/supabase/postgres/blob/develop/ansible/vars.yml#L11

And you will want to bump the minor version on all three of those 

the common.vars.pkr.hcl file will be removed soon/deprecated. Instead, we will generate it at build time, and use the vars in https://github.com/supabase/postgres/blob/develop/ansible/vars.yml#L11 as the "source of truth" for versions of postgres.